schemas/stsci.edu/asdf/0.1.0/core/history_entry.yaml

```diff
@@ -0,0 +1,28 @@
+%YAML 1.1
+---
+$schema: "http://stsci.edu/schemas/yaml-schema/draft-01"
+id: "http://stsci.edu/schemas/asdf/0.1.0/core/history_entry"
+title: |
+  An entry in the file history.
+
+tag: "tag:stsci.edu:asdf/0.1.0/core/asdf"
+type: object
+properties:
+  description:
+    description: |
+      A description of the transformation performed.
+    type: string
+
+  time:
+    description: |
+      A timestamp for the operation.
+    type: string
+    format: date-time
+
+  software:
+    description: |
+      A description of the software that performed the operation.
```

Comment:
True -- I guess the only thing to decide right now is whether each entry in the history list should have a single software entry or a list of software.  Then it would still be up to each program how much detail about its dependencies it wants to include, but it would be slightly more structured than just cramming all of that in the description field.